### ISSUE TITLE
Auto add friend feature

### ISSUE THREAD

**Issue by aysx** [sag-enhanced/sage-issues#23]
### Describe the feature

You can invite friends without spending 5$ using steam friend invite code. That would be nice feature

### Use case

for sending gifts/invites to group etc

**Comment by Le0Developer** [comment]
>using steam friend invite code

Those are one time use. So the main account would need to be into SAGE as well and automatically be logged into and constantly regenerate codes.

Prolly better implemented as post process operation using webhooks or similar

**Comment by aysx** [comment]
Yes, but after generating a new add link, the old link will still be working so would be good to see option to add list of invite links in sage and later you can add third option in "generate" to mass generate invite links for specific account.
That would be awesome to see backward/post process compatibility for "accounts" tab to check accounts which supposed to use that link

**Comment by Le0Developer** [comment]
Popping from a list stored in settings during generation is basically impossible due to the state being in react land.
Basically a billion race conditions are just waiting to happen.

Generating invites on the fly is a LOT better and less complex (since credential caching already exists for steamguard confirmations)

**Comment by Le0Developer** [comment]
This can now be sort of be done in JS using a post generation script: https://sage.party/custom/custom/#creating-a-quick-friend-invite
